hack/e2e-suite/services.sh

```diff
@@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 
 prepare-e2e
 
+echo "Creating test namespace ${NAMESPACE}"
+${KUBECTL} create -f - << __EOF__
+{
+    "kind": "Namespace",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "${NAMESPACE}"
+    },
+    "spec": {
+        "finalizers": [
+            "kubernetes"
```

Comment:
I cannot repro the failure, so I don't know how to fix it properly.  I have
a PR with the original test fix and the not-fixed fix fix.  It works for
me, but not in jenkins.  I don't know how to fix.

On Tue, Jun 2, 2015 at 5:05 PM, Brian Grant <notifications@github.com>
wrote:

> In hack/e2e-suite/services.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9148#discussion_r31583748>
> :
>
> > @@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
> >
> >  prepare-e2e
> >
> > +echo "Creating test namespace ${NAMESPACE}"
> > +${KUBECTL} create -f - << __EOF__
> > +{
> > +    "kind": "Namespace",
> > +    "apiVersion": "v1beta3",
> > +    "metadata": {
> > +        "name": "${NAMESPACE}"
> > +    },
> > +    "spec": {
> > +        "finalizers": [
> > +            "kubernetes"
>
> I don't think this finalizer is needed for a namespace other than default.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9148/files#r31583748>
> .
>